From 8cca7af7e5547f386bd8adfdc46ac1ddf091b10e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?R=C3=A9mi=20Bernon?= <rbernon@codeweavers.com>
Date: Fri, 25 Mar 2022 00:10:04 +0100
Subject: [PATCH 09/16] Revert "fixup! user32/tests: Test a recursive
 activation loop on WM_ACTIVATE."

This reverts commit f3ea0ccdb44906b9c2570aae00d8449d4d4eac80.
---
 dlls/user32/tests/msg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dlls/user32/tests/msg.c b/dlls/user32/tests/msg.c
index 11111111111..11111111111 100644
--- a/dlls/user32/tests/msg.c
+++ b/dlls/user32/tests/msg.c
@@ -10668,7 +10668,7 @@ static LRESULT WINAPI ShowWindowProcA(HWND hwnd, UINT message, WPARAM wParam, LP
 static LRESULT WINAPI recursive_activation_wndprocA(HWND hwnd, UINT message, WPARAM wParam, LPARAM lParam)
 {
     static LONG defwndproc_counter = 0;
-    struct recvd_message msg;
+    struct message msg;
     LRESULT ret;
 
     switch (message)
-- 
2.39.0

